docs/src/components/ComponentDoc/ComponentExample/index.tsx

```diff
@@ -1 +1,2 @@
-export { default, ComponentExampleProps } from './ComponentExample'
+export * from './ComponentExample'
```

Comment:
these changes in exports are necessary to make TS with `--isolatedModules` flag work - this flag ensures that TS compiler uses the same 'file-by-file' processing strategy as Babel